Lib/test/test_turtle.py

```diff
@@ -465,6 +465,19 @@ def test_teleport(self):
             tpen.teleport(-100, -100, fill_gap=fill_gap_value)
             self.assertTrue(tpen.isdown())
 
+class TestRawTurtle(unittest.TestCase):
+    def setUp(self):
+        try:
+            self.screen = turtle.Screen()
+        except turtle.TK.TclError:
+            raise unittest.SkipTest() # cannot instantiate RawTurtle without a screen
+
+    def test_clone(self):
+        rawturtle = turtle.RawTurtle(self.screen)
+        another_turtle = rawturtle.clone()
+        self.assertEqual(another_turtle.currentLineItem, another_turtle.items[-1])
+        self.assertFalse(rawturtle.currentLineItem in another_turtle.items)
```

Comment:
Let's use underscores, and also [`assertNotIn`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertNotIn):
```suggestion
        raw_turtle = turtle.RawTurtle(self.screen)
        another_turtle = raw_turtle.clone()
        self.assertEqual(another_turtle.currentLineItem, another_turtle.items[-1])
        self.assertNotIn(raw_turtle.currentLineItem, another_turtle.items)
```